src/event-helper-listen.js

```diff
@@ -24,32 +31,79 @@
  * @param {!EventTarget} element
  * @param {string} eventType
  * @param {function(!Event)} listener
- * @param {boolean=} opt_capture
+ * @param {Object=} opt_evtListenerOpts
```

Comment:
Just to confirm: do you want to update all callers to only use options struct? Or will you continue to allow `capture` as boolean?